{{ cookiecutter.formal_name }}/briefcase.toml

```diff
@@ -15,4 +15,19 @@ icon.square.96 = "app/src/main/res/mipmap-xhdpi/ic_launcher.png"
 icon.square.144 = "app/src/main/res/mipmap-xxhdpi/ic_launcher.png"
 icon.square.192 = "app/src/main/res/mipmap-xxxhdpi/ic_launcher.png"
 
-# splash.750x1334 = "app/src/main/res/.../launch-750x1334.png"
+splash.160dp_mdpi = "app/src/main/res/drawable/launch.png"
```

Comment:
So - I understand where the "160dp_mdpi" name comes from - but it's fairly opaque unless you know the internal details of Android. 

What would you think about:
* expressing the input list as literal sizes (i.e., `splash.160`, `splash.240`, ... `splash.1440`)
* expressing the input list as sizes, within a "device type" scope (i.e., `splash.smallphone.160`, `splash.phone.160`, `splash.phablet.160`, `splash.tablet.160`, ... `splash.table.1440`)
?

Reply:
I agree that "160dp_mdpi" is a little opaque!

> expressing the input list as literal sizes (i.e., splash.160, splash.240, ... splash.1440)

Honestly, I like this. It has some complications that make me unsure it's worth it, but it still might be the best choice. One upside is that we would be able to re-use images -- you'll see in my initial comment that 160dp @ hdpi is 240px, and 240dp @ mdpi is also 240px. However, this leads to a possible downside -- if we do re-use these images, then the user can't choose a different splash based on screen size/resolution. Hope that makes sense.

> expressing the input list as sizes, within a "device type" scope (i.e., splash.smallphone.160, splash.phone.160, splash.phablet.160, splash.tablet.160, ... splash.table.1440)

I'm open to this in theory. However, in your example, you say `splash.smallphone.160` -- I'd revise that to `splash.smallphone.mdpi`, and then we'd document in the Briefcase docs we expect this to be 160x160. That way, we have a clear distinction between the phone screen size ("smallphone") and the DPI ("mdpi").

Another alternative would be e.g. `splash.160.mdpi`, but now I think we're sort of back at what you didn't like.

Maybe we can discuss over chat exactly what we'd want to go with.

I have also learned that Android gives names to some sizes -- small, normal, large, xlarge -- per this doc: https://developer.android.com/guide/topics/resources/providing-resources#ScreenSizeQualifier . They also call these "legacy size qualifiers" at https://developer.android.com/training/multiscreen/screensizes#TaskUseSizeQuali , making me wonder if they're deprecated? But they don't seem to be; I think it's just that the named sizes predate the more-flexible "smXYZdp" filtering system to filter.